src/crypto/crypto.h

```diff
@@ -102,8 +108,16 @@ namespace crypto {
 
   void hash_to_scalar(const void *data, size_t length, ec_scalar &res);
 
+  /* Normalizes a 16-byte, 13-word seed to a crypto::secret_key
+   */
+  void coerce_valid_sec_key_from(
+    const legacy16B_secret_key &legacy16B_mymonero_sec_seed,
+    secret_key &dst__sec_seed
```

Comment:
yeah this is just what I have taken to doing when using snake case, because it allows one to create tokens within a code name... in camel case I'd use one underscore there instead. Doesn't matter at all to me though :)